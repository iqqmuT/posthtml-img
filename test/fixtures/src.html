<div>
  <img src="https://www.kernel.org/theme/images/logos/tux.png">
  <img src="https://www.kernel.org/theme/images/logos/tux.png" alt="Second">
</div>
