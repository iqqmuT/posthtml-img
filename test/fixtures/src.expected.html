<div>
  <img src="HTTPS://WWW.KERNEL.ORG/THEME/IMAGES/LOGOS/TUX.PNG" width="104" height="120" alt="0_undefined_https://www.kernel.org/theme/images/logos/tux.png">
  <img src="HTTPS://WWW.KERNEL.ORG/THEME/IMAGES/LOGOS/TUX.PNG" alt="1_Second_https://www.kernel.org/theme/images/logos/tux.png" width="104" height="120">
</div>
